build-system/tasks/compile.js

```diff
@@ -130,12 +130,29 @@ function compile(entryModuleFilenames, outputDir,
     define.push('FORTESTING=true');
   }
   if (options.singlePassCompilation) {
-    // TODO(@cramforce): Run the post processing step
-    return singlePassCompile(entryModuleFilenames, {
+    const compilationOptions = {
       define,
       externs: baseExterns,
       hideWarningsFor,
-    }).then(() => {
+    };
+
+    // Add babel plugin to remove unwanted polyfills in esm build
+    if (options.esmPassCompilation) {
+      compilationOptions['babel_plugins'] = [
```

Comment:
can we move this logic for the plugins we want to https://github.com/ampproject/amphtml/blob/master/build-system/build.conf.js ? ideally id like all the logic in one place instead of being scattered

Reply:
done